### PR TITLE
vladmandic/automatic compatibility

### PR DESCRIPTION
**Pull request by ljleb**
This makes the extension work at a basic level in vlad's fork.

### CODE REVIEW

**Comment by ljleb** [comment]
The ui does not adapt to the right height, but it should be possible to fix.

**Review by John-WL — APPROVED** [review]

**Comment by John-WL** [comment]
I don't have the same thing as you:

File: javascript/compute-height.js [hunk 1]
```diff
@@ -1,25 +1,14 @@
...
-}
-
 function computeComfyuiElementHeight() {
     const tab = document.getElementById("tab_comfyui_webui_root");
     const container = document.getElementById("comfyui_webui_container");
     const footerToRemove = document.querySelector('#footer');
-    const dynamicElement = getDynamicElementFromContainer(container);
+    const tabs = document.getElementById("tabs") ?? null;
+    const tab_nav = tabs ? tabs.querySelector(".tab-nav") : null;
+    console.log(tab_nav)
```
Comment (John-WL) [inline javascript/compute-height.js]: can we remove the liog?

**Comment by ljleb** [comment]
The wrong height should be fixed now.

File: javascript/compute-height.js [hunk 2]
```diff
@@ -1,25 +1,14 @@
...
 
-    if(dynamicElement !== null) {
-        const height = dynamicElement.offsetHeight;
-        container.style.height = `calc(100% - ${height-TAB_OFFSET_PADDING}px)`;
+    if(tab_nav !== null) {
```
Comment (John-WL) [inline javascript/compute-height.js:8]: Do you think
```js
if(tab_nav) {
```
would be better?

Comment (ljleb) [reply javascript/compute-height.js:8]: In js `if(reference)` is the same as `!= undefined && != null` or `!== undefined && !== null`? or something else? Just to be safe I went the explicit route.

**Pull request merged**